eval/src/tests/eval/tensor_lambda/tensor_lambda_test.cpp

```diff
@@ -77,14 +78,14 @@ void verify_impl(const vespalib::string &expr, const vespalib::string &expect) {
     verify_impl<T>(expr, expect, [](const T*){});
 }
 
-void verify_generic(const vespalib::string &expr, const vespalib::string &expect,
-                    EvalMode expect_eval_mode)
-{
-    verify_impl<DenseLambdaFunction>(expr, expect,
-                                     [&](const DenseLambdaFunction *info)
-                                     {
-                                         EXPECT_EQUAL(info->eval_mode(), expect_eval_mode);
-                                     });
+void verify_not_optimized(const vespalib::string &expr, const vespalib::string &expect) {
```

Comment:
I like the verify_generic name better here

Reply:
done.